style.css

```diff
@@ -663,6 +663,10 @@
     padding-bottom: 25px;
     gap: 12px;
   }
+
+  label {
+    display: none;
+  }
 }
 
```

Comment:
- [ ] Please make sure that you fix the accessibility of the `see project` color contrast. You can use [this aditus tool](https://www.aditus.io/button-contrast-checker/) to help to check the colour contrast of the project.
the color contrast ratio is failing as you can see from the screenshot below.
  ![Screenshot from 2022-12-13 21-46-43](https://user-images.githubusercontent.com/92303891/207452613-b10e9be5-d478-44cf-8f86-a3aba001b9c1.png)

- [ ] You can adjust the color of the text and the background to make them pass so that some people cannot read the text if there is not sufficient contrast between the text and background, for example, light gray text on a light background. :smile: